sdk/iot/core/tests/cmocka/main.c

```diff
@@ -41,6 +62,21 @@ int main()
     cmocka_unit_test(az_iot_sas_token_generate_sas_token_overflow_fails),
     cmocka_unit_test(az_iot_sas_token_generate_succeeds),
     cmocka_unit_test(az_iot_sas_token_generate_with_keyname_succeeds),
+
+    //Telemetry Unit Tests
```

Comment:
>// [](start = 4, length = 2)

// Telemetry

Reply:
Space between // and comment

---
In reply to: [389176169](https://github.com/CIPop/azure-sdk-for-c/pull/1#discussion_r389176169) [](ancestors = 389176169)